rnaseq.snakefile

```diff
@@ -2,7 +2,7 @@ import os
 import pandas as pd
 from lcdblib.snakemake import helpers, aligners
 from lcdblib.utils import utils
-import common
+from lib import common
 
 shell.prefix('set -euo pipefail; ')
```

Comment:
Good call, for now I'll do: 

```python
TMPDIR = tempfile.gettempdir()                                                                                                                                                                                                                                           
JOBID = os.getenv('SLURM_JOBID')                                                                                                                                                                                                                                         
if JOBID:                                                                                                                                                                                                                                                                
    TMPDIR = os.path.join('/lscratch', JOBID)                                                                                                                                                                                                                            
shell.prefix('set -euo pipefail; export TMPDIR={};'.format(TMPDIR))
```
 might be useful to add a `get_temp_dir()` to lcdblib or common.py